libbeat/cmd/instance/beat.go

```diff
@@ -1072,20 +1073,28 @@ func (b *Beat) configure(settings Settings) error {
 		return fmt.Errorf("error setting timestamp precision: %w", err)
 	}
 
+	// TODO: Should be eventually removed. This is not removed so that we do not break the existing dependency on global loggers
```

Comment:
Perhaps we could already have an issue opened for that and have this TODO to point to it

Reply:
I believe the issue this PR refers to is sufficient https://github.com/elastic/beats/pull/43356

This change would be a part of it eventually.